LabExT/View/Movement/CoordinatePairingsWindow.py

```diff
@@ -161,6 +172,24 @@ def __setup__(self) -> None:
                 frame, self.mover.output_calibration).pack(
                 side=TOP, fill=X)
 
+        if self.experiment_manager:
+            shortcuts_frame = CustomFrame(self._main_frame)
+            shortcuts_frame.pack(side=TOP, fill=X, pady=5)
+
+            search_for_peak_button = Button(
+                shortcuts_frame,
+                text="Perform Search for Peak...",
+                command=self._on_perform_search_for_peak)
+            search_for_peak_button.pack(
+                side=RIGHT, fill=Y, pady=5, expand=0)
+
+            live_viewer_button = Button(
+                shortcuts_frame,
+                text="Open Live Viewer...",
+                command=self._on_open_live_viewer)
```

Comment:
Same here, reuse the function `self.experiment_manager.main_window.open_peak_searcher` with no need to re-code logic in this class.